### ISSUE TITLE
ng dynamic breadcrumb is not compatible with angular 15

### ISSUE THREAD

**Issue by FaizanBarmare** [rajaramtt/ng7-dynamic-breadcrumb#48]
Angular 15 has been rollout in November 2022 but still there are no version upgrade for angular 15. We are getting an issues of peer dependencies with @angular/common 15 version.

Will you be providing any latest updates or not for angular 15? If yes then expected time would be?

**Comment by bjorn-einar-bjartnes-4ss** [comment]
I added a PR now to upgrade to angular 16. 
A similar PR can be made with angular 15, I reserved the version number 9 for it https://github.com/rajaramtt/ng7-dynamic-breadcrumb/pull/49

**Comment by craigneasbey** [comment]
Hi @bjorn-einar-bjartnes-4ss , please correct my understanding, looking at https://github.com/rajaramtt/ng7-dynamic-breadcrumb/pull/51/files, "^16.1.8" does not include Angular 15.  Our requirement is for Angular 15.

Just re-reading this, you are saying that a separate 15 PR would be required.
```
ng update @angular/animations@15 @angular/cdk@15 @angular/common@15 @angular/compiler@15 @angular/core@15 @angular/forms@15 @angular/material@15 @angular/platform-browser@15 @angular/platform-browser-dynamic@15 @angular/router@15 @angular/compiler-cli@15 @angular/language-service@15 @angular-devkit/build-angular@15 @angular/cli@15
```

Thanks

**Comment by bjorn-einar-bjartnes-4ss** [comment]
Yes, that is why I skipped the version number 9, to allow for an Angular 15 version.

**Comment by craigneasbey** [comment]
I added #52 .  Seem to work fine.

**Comment by himanshujjj1** [comment]
@craigneasbey @rajaramtt  @bjorn-einar-bjartnes-4ss  I am new to this. How we can add this update to our project? Could you please help me with the steps.
